participants/yituyun.md

```diff
@@ -0,0 +1,67 @@
+# DBC-AI-Distributed-Cloud-Storage
+
+Before submitting a PR containing your team's project information:
+- Make sure you rename this file to contain your project's name
+- Respond to add your response in each section below
+
+## (0) Project Name and Team Slack Handles
+
+- Project Name:
+    - *yituyun 忆图云*
+
+- Team Slack Handles:
+    - *litai686@qq.com*
+
+## (1) Brief Project Description
+
+*An open image search library, where everyone is a picture provider and contributor, is stored on the Filecoin Network.*
+
+## (2) Link to Project UI
+
+*https://github.com/litai686/imgSearch.git*
+*http://f.etu1314.com*
+
+## (3) What does your application/UI do?
+
+*An open image search library, where everyone is a picture provider and contributor, is stored on the Filecoin Network.*
+
+## (4) If your project is using a curated dataset, which one are you using?
+
+*We didn't use the data.*
```

Comment:
You may want to consider an open image set like Flikr Commons to start populating data faster .